app/src/Header.js

```diff
@@ -16,6 +16,11 @@ const Header = (props) => {
       >
         state
       </a>
+      <div className="right menu">
+        <a href="logout" className="ui item">
```

Comment:
Please explain the functioning here, as to how the logout method in logout.js is triggered on clicking this?